doc/developer/requirements.md

```diff
@@ -0,0 +1,129 @@
+# Install requirements
+
+-   [Git](https://git-scm.com/)
+-   [Node.js](https://nodejs.org/)
+-   [Yarn](https://yarnpkg.com/)
+-   [Docker](https://www.docker.com/)
+-   [Cozy Stack in Docker](#set-up-a-cozy-stack)
```

Comment:
Docker & a running cozy-stack are not necessary to build the app from source.

What about separating `doc/build_from_source.md` (including build requirements & process description) and `doc/developer/cozy_stack_setup.md` (including both with & without docker options)?